### ISSUE TITLE
Define business approval flow breakdown

### ISSUE THREAD

**Issue by LeonJLobo** [Lieshaz/VentureHere#1]
Raise questions with the business approval and breakdown flows.

**Comment by Lieshaz** [comment]
@LeonJLobo , should they be two separate issues?

**Comment by LeonJLobo** [comment]
@Lieshaz Yes, I thought we list all flows/concerns and then narrow down to MVP

**Comment by LeonJLobo** [comment]
@Lieshaz I’m listing the open questions on the business plan here.

1. How is a milestone achievement evaluated?
2. How do we convince the entrepreneur of a guarantee of funds?
3. How do we guarantee refunds to investors?
4. How do we draw a legal contract automatically and online?
5. How does equity translate to shares when company goes public?
6. When does the flow end for the entrepreneur and user?
7. How do we prevent idea plagiarism?

**Comment by LeonJLobo** [comment]
Proposal flow for wave 1

1. Entrepreneur posts idea with milestone and payout per template.
2. Investor browses through multiple idea options based on interests and decides to invest in idea. Investor has seed option, milestone investing option.
3. Official contract between both parties drawn up and signed automatically.
4. Seed money provided directly. Money held at secure third party
5. Entrepreneur hits milestone 1 within deadline and passes evaluation.
6. Secure funding released to entrepreneur.
7. Entrepreneur misses milestone 2 evaluation, can improve offer to entice investors to stay.
8. Evaluation report sent to investor with option to opt out or stay.
9. Only stay money transferred to entrepreneur, rest is refunded.
10. Once milestones are complete, flow ends.

**Comment by LeonJLobo** [comment]
Market pricing proposal

1. Charge entrepreneurs a registration fee. Other option is entrepreneur pays in equity(riskier).
2. Do not charge investors, they anyways provide seed money. This will incentivize more investors to hop on the platform, which leads to more entrepreneurs.

In later iterations

3. Incentivize platform for premium investors. Premium is defined by prior money invested or by reputation. Incentive can be early access or subscription money.
4. Provide investors with a paid option(nominal monthly fee) which allows subscribing to premium investors. Premium investors and us get a cut.

**Comment by LeonJLobo** [comment]
Future waves

1. Provide relevant searches so that user’s prior investing and interests pull in similar results on their home page.
2. Show investing trends.
3. Allow for investors to sell their investments for a profit/loss mid milestone (only paid users can sell?)
4. Rate investment success to score users based on seed money and score.
